mappings/net/minecraft/util/math/MathHelper.mapping

```diff
@@ -277,5 +280,16 @@ CLASS net/minecraft/class_3532 net/minecraft/util/math/MathHelper
 		ARG 0 x
 		ARG 1 y
 		ARG 3 z
+	METHOD method_34939 clamp (BBB)B
+		ARG 0 value
+		ARG 1 min
+		ARG 2 max
+	METHOD method_34947 parseDouble (Ljava/lang/String;D)D
+		ARG 0 string
+		ARG 1 fallback
+	METHOD method_34951 cumulativeProportions ([D)[D
```

Comment:
What does this method do?

Reply:
This method calculates `(values[0] + ... + values[i]) / (values[0] + ... + values[values.length - 1])` for each `i`.